packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -15,15 +15,15 @@ const data: LandingTemplateSchema = {
   sections: [
     {
       type: 'Generic',
-      anchorLink: 'support-policy',
-      title: 'Support Policy',
+      anchorLink: 'compatibility-policy',
+      title: 'Compatibility Policy',
       sectionContent: '',
       sectionNotice: [
         {
-          title: 'End of Support Plan',
+          title: 'End of Compatibility with POS Plan',
```

Comment:
This sounds like it's an end of compatibility with certain POS Plans.
```suggestion
          title: 'End of Compatibility Plan',
```